components/frontend/src/metric/MetricDetails.test.jsx

```diff
@@ -12,12 +12,14 @@ import * as source_api from "../api/source"
 import { DataModel } from "../context/DataModel"
 import { EDIT_ENTITY_PERMISSION, EDIT_REPORT_PERMISSION, Permissions } from "../context/Permissions"
 import { expectNoAccessibilityViolations } from "../testUtils"
+import * as toast from "../widgets/toast"
```

Comment:
yikes `import *`, well at least it's test code

Reply:
This is the pattern we use for mocking modules. But it may well be an outdated pattern. I don't see it in the Jest docs: https://jestjs.io/docs/mock-functions. Changing it is out of scope for this issue, but I'll open a separate issue.